migrations/Version20201007091517.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20201007091517 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema) : void
```

Comment:
Codacy found an issue: [Avoid using short method names like Version20201007091517::up(). The configured minimum method name length is 3.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6297993)